releasenotes/notes/fix-path2d-mpl3.4-b1af3a23b408d30a.yaml

```diff
@@ -0,0 +1,14 @@
+---
+fixes:
+  - |
+    Fixed an issue where plotting Bloch spheres could cause an ``AttributeError``
+    to be raised in Jupyter or when trying to crop figures down to size with
+    Matplotlib 3.4.  For example, the following code would previously crash with a
```

Comment:
What about 3.3? Iirc the min supported version for mpl is 3.3 do things still work with it?

Reply:
Good point - I just checked, and it crashed on MPL 3.3 as well before this patch, and is fixed after it.  This is the bug that you and I knew about right before Terra 0.19 went out, but couldn't reproduce.